it/uk/gov/hmrc/traderservices/controllers/TraderServicesFrontendISpec.scala

```diff
@@ -16,6 +16,7 @@ import uk.gov.hmrc.traderservices.stubs.{JourneyTestData, TraderServicesStubs}
 import uk.gov.hmrc.traderservices.support.{ServerISpec, TestJourneyService}
 
 import scala.concurrent.ExecutionContext.Implicits.global
+import uk.gov.hmrc.traderservices.models.ExportQuestions
 
 class TraderServicesFrontendISpec
```

Comment:
Ran the IT from the command line and there were 5 failures for this Spec. I'm guessing they are due to the new paths etc